src/geo/ui/infowindow.js

```diff
@@ -327,39 +363,39 @@ var Infowindow = View.extend({
     }
 
     if (!this._isValidURL(url)) {
-      $shadow.hide();
       log.info("Header image url not valid");
       return;
     }
 
-    // create the image
-    $img = $("<img />").attr("src", url);
+    $img = $("<img class='CDB-infowindow-media-item' />").attr("src", url);
+
     $cover.append($img);
 
     $img.load(function(){
-      var w  = $img.width();
-      var h  = $img.height();
+      var w = $img.width();
+      var h = $img.height();
+
       var coverWidth = $cover.width();
       var coverHeight = $cover.height();
-      var calculatedHeight = h / (w / coverWidth);
+
       var ratio = h / w;
+
       var coverRatio = coverHeight / coverWidth;
-      var styles = {
-        width: coverWidth,
-        top: "50%",
-        position: "absolute",
-        "margin-top": -1*parseInt(calculatedHeight, 10)/2
-      };
+
+      var styles = {};
 
       // Resize rules
-      if ( w > coverWidth && h > coverHeight) { // bigger image
-        if ( ratio < coverRatio ) {
+      if (w > coverWidth && h > coverHeight) { // bigger image
+        if (ratio < coverRatio) {
           styles = { height: coverHeight };
         }
       }
-
+      
       $img.css(styles);
-      $img.fadeIn(300);
+      $cover.css({ height: h - self.options.hookHeight });
+      $img.fadeIn(self.options.imageTransitionSpeed);
+
+      self._loadImageHook(url);
```

Comment:
Shouldn't we load/set hook image when there is only image + title?
cc @piensaenpixel